src/main/webapp/js/finscholar/collegepageview/collegepageview.js

```diff
@@ -16,15 +16,18 @@
 
 goog.module('finscholar.collegepageview');
 
-const GoogDom = goog.require('goog.dom');
+const {SinglePageView} = goog.require('finscholar.singlepageview');
 const {loadCollegeData} = goog.require('datahandlers.collegepage');
 
 /** Class for the college page view. */
-class CollegePageView {
-  constructor() {}
+class CollegePageView extends SinglePageView {
+
+  constructor() {
+    super(undefined, undefined);
+  }
 
   /** Render the college page. */
-  async renderView(element) {
+  async renderView(element, id) {
```

Comment:
id is never used?

Reply:
Oh the loadCollegeData method doesn't query by id in this version. This collgepageview class is just a framework for #74 and #73